src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //================================================
+  //MAIN CONTAINER
+  //================================================
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(600px);
+    padding-top: $spacing-3xs; // allow space for focus styles
+
+    .#{$prefix}--search-input {
+      background-color: transparent;
+    }
+  }
+
+  //================================================
+  //TABLE HEADER
+  //================================================
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    color: $text-01;
+    @include font-size('20');
+    font-weight: 400;
+    padding: 22px 0 0 $spacing-md;
+    position: relative;
+    top: rem(21px);
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    position: relative;
+    left: rem(17px);
+    top: rem(-20px);
+    @include font-size('14');
+    color: $text-02;
+  }
+
+  //================================================
+  //DATA TABLE - DEFAULT STYLES
+  //================================================
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+
+    [class*='col'] {
```

Comment:
Is this for focus styles on table cells? This might be a bigger discussion around focus management in tables but I'm not sure if we want to do this

Reply:
I added this a while back because a different PR added some styles that were affecting this. https://github.com/IBM/carbon-components/pull/1650/files#diff-33ebb84c797d2e8a44086641297274e6R182 I removed it now since it's been fixed.